examples/src/index.js

```diff
@@ -131,7 +131,7 @@ function Demo() {
             />
           </Grid.Column>
 
-          <Grid.Column>
+          <Grid.Column className="ap-richtext-editor">
```

Comment:
this class name is a little confusing since we also have `ap-rich-text-editor` which is only different by a hyphen in the src code. Can you make this one `demo-rich-text-editor` instead and then update the `examples/src/index.css` file as well?

Reply:
Sure! Should I just change that with the test update commit?